site/app/templates/admin/library/LibraryFilePath.twig

```diff
@@ -0,0 +1,16 @@
+<form class="form-signin" action="{{ filepath_submit_url }}" method="post">
+    <div class="option row">
+        <div class="option-desc col-md-6">
+            <label for="instructions_url">
+                <div class="option-title">File Path</div>
+                <div class="option-alt">What is the file path to the gradeable?</div>
```

Comment:
Is this taking a file path to a zip file already on the server? Do I need to make backend support for that?

Reply:
Yeah. I think Course Materials > Upload Course Materials does something like this.